genisys/main.py

```diff
@@ -66,3 +66,32 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+'''
+import needed modules,
+create new module with options/configs from other branches.
+'''
+def run():
+
+    #netplan.py
+    import modules.netplan as net
+
+    #get settings for functions
+    netplan = net.Netplan()
+    netLocation = net.install_location()
+    netgenerate = net.generate()
+
+    #base.py
+    import modules.base as base
+
+    #get settings for functions
+    install = base.install(__path__)
+    validate = base.validate()
+
+    #configParser.py
+    import genisys.configParser as cp
```

Comment:
The run function shouldn't need to make it's own instance of the configParser. The main function will parse the config based on certain command line options then pass that instance of the config to the run function.